### ISSUE TITLE
Malformed arrow function parameter list, after minifying

### ISSUE THREAD

**Issue by GilianJuice** [trullock/NUglify#142]
I have found a problem with the minifier, if I minify the code below:
```javascript
let isReady = false;
const core = {
    utils: {
        noop() { }
    }
};

const test = {
    ready: (onReady = core.utils.noop) => {
        if (isReady) {
            onReady();
        } else {
            document.addEventListener("DOMContentLoaded", onReady);
        }
    }
};
```

This will have the following result:
```javascript
let isReady=!1;const core={utils:{noop(){}}},test={ready:n=core.utils.noop=>{isReady?n():document.addEventListener("DOMContentLoaded",n)}};
```
Which will cause the error `Uncaught SyntaxError: Malformed arrow function parameter list`.

This is because there are some brackets missing. If you change the above code to this:
```javascript
let isReady=!1;const core={utils:{noop(){}}},test={ready:(n=core.utils.noop)=>{isReady?n():document.addEventListener("DOMContentLoaded",n)}};
```
That will fix the problem (note the extra brackets I added around `n=core.utils.noop`).

**Comment by trullock** [comment]
Good spot, this is an issue with default parameters

**Comment by trullock** [comment]
Quick workaround is to add a 2nd parameter, e.g.

`ready: (onReady = core.utils.noop, undefined) => {`

This will be fixed in 1.9.3 shortly

**Comment by GilianJuice** [comment]
The fix works, thanks!
